service/src/main/kotlin/fi/espoo/evaka/reports/StartingPlacementsReport.kt

```diff
@@ -66,6 +67,16 @@ private fun Database.Read.getStartingPlacementsRows(year: Int, month: Int): List
 
     return createQuery(sql)
         .bind("range", FiniteDateRange.ofMonth(year, Month.of(month)))
-        .mapTo<StartingPlacementsRow>()
+        .map { mapper ->
+            StartingPlacementsRow(
+                careAreaName = if (mapper.mapColumn("is_private_service_voucher_daycare")) "palvelusetelialue" else mapper.mapColumn("care_area_name"),
+                childId = ChildId(mapper.mapColumn("child_id")),
+                firstName = mapper.mapColumn("first_name"),
+                lastName = mapper.mapColumn("last_name"),
+                dateOfBirth = mapper.mapColumn<String>("date_of_birth").let { LocalDate.parse(it) },
+                ssn = mapper.mapColumn("ssn"),
+                placementStart = mapper.mapColumn<String>("placement_start").let { java.time.LocalDate.parse(it) },
```

Comment:
This could just be `placementStart = mapper.mapColumn("placement_start")`.

Reply:
Initially Idea complained about the types, I found how we hande that case elsewhere, now that I reverted back to no special handling, no complaints anymore 🤷 . Anyway, fixed, thanks again!